autoload/caw.vim

```diff
@@ -17,9 +18,17 @@ function! caw#keymapping_stub(mode, action, method) abort
     normal! ^
   endif
 
+  " When integration == 'context_filetype'
+  "   Load comment string from the context filetype
+  " When integration == 'ts_context_commentstring'
+  "   Load comment string from ts_context_commentstring plugin
+  " When integration == ''
+  "   No additional setup will be done
+  let integration = s:get_integrated_plugin()
+
   " Context filetype support.
   " https://github.com/Shougo/context_filetype.vim
-  if s:installed_context_filetype
+  if integration ==# 'context_filetype'
```

Comment:
I'm not 100% sure how Shougo/context_filetype.vim works, but I think that a user might use both `context_filetype.vim` and `nvim-ts-context-commentstring` (in different files). Some languages like vimscript don't have a treesitter parser, so using `context_filetype.vim` for those makes sense. Right now, it looks like only one of those integrations would work.